jekyll/_cci2/github-integration.adoc

```diff
@@ -0,0 +1,279 @@
+
+You can read more about these settings and how to configure them on https://docs.github.com/en/organizations/restricting-access-to-your-organizations-data/enabling-oauth-app-access-restrictions-for-your-organization[GitHub].
+
+NOTE: If you enable these restrictions on an organization for which CircleCI has been running builds, CircleCI will stop receiving push event hooks from GitHub, and will not build new pushes. API calls will also be denied, causing, for instance, re-builds of old builds to fail the source checkout. To get CircleCI working again, you will need to grant access to the CircleCI application.
+
+=== How to re-enable CircleCI for a GitHub organization
+
+This section describes how to re-enable CircleCI after enabling third-party application restrictions for a GitHub organization. Go to https://github.com/settings/connections/applications/78a2ba87f071c28e65bb[GitHub Settings], and in the **Organization access** section, you will have the option to request access if you are not an admin, or grant access if you are an admin.
+
+==== Non-admin member workflow
+
+- If you are member of a GitHub org (not an admin), click the **Request** button and a message will be sent to an admin of your organization. An admin will have to approve the request.
+- Click **Request approval from owners** to send an email to your organization’s owners.
+- While waiting for approval, you will see **Access request pending** next to your organization’s name.
+- If CircleCI has been approved by your organization, you will see a checkmark next to your organization’s name.
+
+==== Admin owner workflow
+
+- If you are an owner of your organization (an admin), you may grant access to CircleCI by clicking on the **Grant** button.
+- You may be asked to confirm your password in order to authorize our app.
+- Once you’ve approved CircleCI, you will see a checkmark next to your organization’s name.
+
+After access is granted, CircleCI should behave normally again.
+
+== Rename orgs and repositories
+
+If you find you need to rename an org or repo that you have previously hooked up to CircleCI, the best practice is to follow these steps:
+
+1. Rename org/repo in GitHub.
+1. Head to the CircleCI application, using the new org/repo name, for example, `app.circleci.com/pipelines/github/<new-org-name>/<project-name>`.
+1. Confirm that your plan, projects and settings have been transferred successfully.
+1. You are then free to create a new org/repo with the previously-used name in GitHub, if desired.
+
+**Note**: If these steps are not followed, you might lose access to your org or repo settings, including **environment variables** and **contexts**.
```

Comment:
```suggestion
NOTE: If these steps are not followed, you might lose access to your org or repo settings, including **environment variables** and **contexts**.
```

Reply:
Also to add a Next Steps section at the end of these integration pages we could signpost people to the project settings overview? https://circleci.com/docs/2.0/settings/ - just off the top of my head